ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Assertions.scala

```diff
@@ -69,41 +69,13 @@ object Assertions {
     assertSameElements(actual, Seq.empty)
   }
 
-  case class ExtendedAsserts(
-      exceptionMessageSubstring: Option[String] = None,
-      checkDefiniteAnswerMetadata: Boolean = false,
-      additionalErrorAssertions: Throwable => Unit = _ => (),
-  )
-
-  def assertGrpcErrorOneOf(
```

Comment:
Removing previously introduced `extendedChecks` powers of `assertGrpcErrorOneOf` that didn't handle `checkDefiniteAnswerMetadata` depending on the `ErrorCode`.